cli/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.6.1
+
+_Released 03/1/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Fixed an issue where cookies were being duplicated with the same hostname, but with a prepended dot. Fixes [#25174](https://github.com/cypress-io/cypress/issues/25174).
```

Comment:
@emilyrohrbough I broke this out into separate bugfixes but the changelog job isn't too happy about it 😅

Reply:
boooo 😅 I will need to update this when I can find time.

Reply:
reverted in 856781222f